src/GZip.jl

```diff
@@ -219,12 +219,12 @@ let _zlib_h = Libdl.dlopen(_zlib)
         const _gzopen = :gzopen64
         const _gzseek = :gzseek64
         const _gztell = :gztell64
-        #_gzoffset = :gzoffset64  ## not implemented
+        const _gzoffset = :gzoffset64
```

Comment:
It's quite possible this wasn't implemented because it didn't exist in earlier versions of zlib.  At the least, the version on travis doesn't seem to find it.